wsd/COOLWSD.cpp

```diff
@@ -2207,18 +2212,19 @@ void COOLWSD::innerInitialize(Application& self)
     // Initialize the config subsystem too.
     config::initialize(&config());
 
-    bool bindMount = getConfigValue<bool>(conf, "mount_jail_tree", true);
+    IsBindMountingEnabled = getConfigValue<bool>(conf, "mount_jail_tree", true);
 #if CODE_COVERAGE
     // Code coverage is not supported with bind-mounting.
-    if (bindMount)
+    if (IsBindMountingEnabled)
     {
         LOG_WRN("Mounting is not compatible with code-coverage. Disabling.");
-        bindMount = false;
+        IsBindMountingEnabled = false;
     }
 #endif // CODE_COVERAGE
+        IsBindMountingEnabled = false;
```

Comment:
IsBindMountingEnabled is always false

Reply:
Good catch! This was for testing, will remove it.